### PR TITLE
py3-build versioned python

### PR DESCRIPTION
**Pull request by pnasrat**
Move py3-build to use versioned python

### CODE REVIEW

File: py3-build.yaml [hunk 1]
```diff
@@ -31,31 +34,53 @@ pipeline:
...
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr
+          mv ${{targets.contextdir}}/../py${{range.key}}-${{vars.pypi-package}}/usr/bin ${{targets.contextdir}}/usr
 
-  - uses: strip
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
```
Comment (vaikas) [inline py3-build.yaml:71]: Should this be depending on the -bin version which pulls in the binaries, and then transivitely the libs?

Comment (vaikas) [reply py3-build.yaml:71]: Asking mainly because I'm trying to document the new way of doing things 🤣

File: py3-build.yaml [hunk 2]
```diff
@@ -31,31 +34,53 @@ pipeline:
...
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+
+test:
+  pipeline:
+    - runs: python3.12 -c "import ${{vars.pypi-package}}"
```
Comment (vaikas) [inline py3-build.yaml:77]: ❤️

File: py3-build.yaml (hunk 1 above)
Comment (pnasrat) [reply py3-build.yaml:71]: Supported is really for us to build packages and we can just use  `python -m build` vs the binary build (which I don't think is heavily used)

**Review by vaikas — APPROVED** [review]

**Pull request merged**